infrastructure/templates/public-api/components/alerts/baseCpuPercentageAlert.bicep

```diff
@@ -0,0 +1,38 @@
+import { Severity } from 'types.bicep'
+
+@description('Names of the resources that these alerts are being applied to.')
+param resourceNames string[]
+
+@description('Names of the resources that these alerts are being applied to.')
+param resourceType string
```

Comment:
Could create and use a new `CpuResourceType` that is a union of the specific resources with CPUs (rather than every resource type). This would help with auto-completion and general type safety.